pkg/asset/installconfig/vsphere/client.go

```diff
@@ -63,12 +63,17 @@ func CreateVSphereClients(ctx context.Context, vcenter, username, password strin
 	restClient := rest.NewClient(c.Client)
 	err = restClient.Login(ctx, u.User)
 	if err != nil {
-		c.Logout(context.TODO())
+		logoutErr := c.Logout(context.TODO())
+		if logoutErr != nil {
+			err = logoutErr
+		}
 		return nil, nil, nil, err
 	}
 
 	return c.Client, restClient, func() {
+		//nolint:all
 		c.Logout(context.TODO())
+		//nolint:all
```

Comment:
Can't you disable just a specific linter instead of all of them? `//nolint:<linter name>`
Otherwise we might be ignoring valid linter warnings.

Reply:
will fix this

Reply:
lol, the linter is funny, it randomly finds issues then turns around and misses it the next time.

Reply:
I wonder if setting https://github.com/openshift/installer/blob/master/.golangci.yaml#L17 to `true` would help.